frontend-react/src/pages/Login.tsx

```diff
@@ -15,7 +15,12 @@ export const Login = () => {
 
     const navigate = useNavigate();
     useEffect(() => {
-        if (authState && authState.isAuthenticated) navigate("/");
+        if (authState && authState.isAuthenticated) {
+            // TODO (router-refactor): Dynamically navigate users
```

Comment:
Will address this on shadow branch final pass before PR opens.

Reply:
can we remove the comment, or is there still something to do here?

Reply:
bumping this comment - should we make a new card for re-implementing this?

Reply:
This was added to hopefully prevent someone who's logged in from landing on `/login` but breaks the okta redirect handler, so I removed it.